### PR TITLE
refactor: configs

### PR DESCRIPTION
**Pull request by codekow**

### CODE REVIEW

**Comment by codekow** [comment]
@channolanp

How do you feel about these changes?

**Comment by codekow** [comment]
I reorganized to allow more flexibility with overlays

File: components/app-configs/mlde-workspace-template/base/kustomization.yaml [hunk 1]
```diff
@@ -1,17 +1,15 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
-namespace: <replace me>
```
Comment (channolanp) [inline components/app-configs/mlde-workspace-template/base/kustomization.yaml:4]: Do we actually want to remove the namespace for the entire kustomize context?

Comment (channolanp) [reply components/app-configs/mlde-workspace-template/base/kustomization.yaml:4]: Nevermind, I should have read the rest of the changes

**Review by channolanp — APPROVED** [review]
LGTM

File: components/app-configs/mlde-workspace-template/base/kustomization.yaml (hunk 1 above)
Comment (codekow) [reply components/app-configs/mlde-workspace-template/base/kustomization.yaml:4]: @channolanp You make a good point... moved the namespace to the overlay so the base is more generic

Comment (codekow) [reply components/app-configs/mlde-workspace-template/base/kustomization.yaml:4]: Then someone can define the namespace and only apply the base if they want.

**Pull request merged**